spec/ruby/core/module/const_source_location_spec.rb

```diff
@@ -233,5 +233,17 @@
       line = ConstantSpecs::CONST_LOCATION
       ConstantSpecs.const_source_location('CONST_LOCATION').should == [file, line]
     end
+
+    ruby_bug("#20188", ""..."3.4") do
```

Comment:
```suggestion
    ruby_version_is "3.4" do # https://bugs.ruby-lang.org/issues/20188
```
`ruby_bug` is too much here, i.e. other Rubies shouldn't necessarily change this before they target 3.4, in fact it could maybe even be incompatible for some programs (likely not, but who knows).
In short `ruby_bug` is almost only to avoid CRuby segfaults.

Reply:
My interpretation is that this is a bug, hence will be backported, but yes, perhaps it's not one, and perhaps it won't even be accepted as a feature. 

I'll make this change depending on how it goes with the issue.

Reply:
It's true that if this would be decided to be backported then using `ruby_bug` is appropriate.